src/main/target/KIWIF4/README.md

```diff
@@ -0,0 +1,16 @@
+##KIWI F4
```

Comment:
Thanks for the description. Can you please copy it into the 'Boards' list on the wiki (https://github.com/betaflight/betaflight/wiki)? It's easier to find there for new users.

(Love the 'kiwi' name. :-P)